### ISSUE TITLE
update ADNI clinical data 

### ISSUE THREAD

**Issue by AyaRabea** [aramis-lab/clinica#1344]
Thanks for the greet work

Currently, the ADNI website has been updated, and after the update, I can't find all the clinical data CSV files—I only found the ADNIMERGE file.
 Can you please help?

**Comment by AliceJoubert** [comment]
Hello,

We indeed noticed that the ADNI platform has been updated recently. However, it seems to be unstable with new modifications happening regularly and no communication from ADNI's side. Both the converter and the user instructions need to be adapted to these changes but we will be able to work on it only when ADNI downloading platform is stabilized. As of now, it is difficult to know what the modifications performed by ADNI would break in the converter, so it is not just a matter of finding the files on the platform.

I would advise to use previous downloaded versions of the clinical data for ADNI if you have them.

Best,
